Mathlib/Topology/UnitInterval.lean

```diff
@@ -69,12 +61,6 @@ lemma univ_eq_Icc : (univ : Set I) = Icc (0 : I) (1 : I) := Icc_bot_top.symm
 @[simp, norm_cast] theorem coe_pos {x : I} : (0 : ℝ) < x ↔ 0 < x := Iff.rfl
 @[simp, norm_cast] theorem coe_lt_one {x : I} : (x : ℝ) < 1 ↔ x < 1 := Iff.rfl
 
-instance : Nonempty I :=
-  ⟨0⟩
```

Comment:
Where did this one go?

Reply:
We have a generic `[Zero α] : Nonempty α`. Should I bring it back as a shortcut instance?